deltachat-ios/ViewModel/ChatListViewModel.swift

```diff
@@ -184,6 +184,7 @@ class ChatListViewModel: NSObject {
         let chat = dcContext.getChat(chatId: chatId)
         let isArchivedBefore = chat.isArchived
         if !isArchivedBefore {
+            dcContext.marknoticedChat(chatId: chatId)
             NotificationManager.removeNotificationsForChat(dcContext: dcContext, chatId: chatId)
         }
         dcContext.archiveChat(chatId: chatId, archive: !isArchivedBefore)
```

Comment:
`archiveChat()` already makes things as being notices. so, the issue is more that removeNotificationsForChat() is called _before_ `archiceChat()`, i am on that anyway, i will do a commit.